packages/components/upload/src/upload-content.vue

```diff
@@ -74,6 +74,7 @@ const uploadFiles = (files: File[]) => {
     onStart(rawFile)
     if (autoUpload) upload(rawFile)
   }
+  Object.assign(fileList, files)
```

Comment:
I think we should change the logic of `handleStart` to fix it.
https://github.com/element-plus/element-plus/blob/36b6106effd417003747c2ba987a3589ade2172a/packages/components/upload/src/use-handlers.ts#L103

Reply:
we can use `Object.assign(uploadFiles.value, [...uploadFiles.value, uploadFile])` to fix it

Reply:
Correct it `Object.assign(uploadFiles.value, [uploadFile])`

Reply:
we can use Object.assign(uploadFiles.value, [uploadFile]) to fix it

Reply:
we can use Object.assign(uploadFiles.value, [...uploadFiles.value, uploadFile]) to fix it